ragna/deploy/_core.py

```diff
@@ -79,8 +82,14 @@ def server_available() -> bool:
         app.include_router(make_api_router(engine), prefix="/api")
 
     if ui:
-        panel_app = make_ui_app(engine)
-        panel_app.serve_with_fastapi(app, endpoint="/ui")
+        ui_app = make_ui_app(engine)
+        panel.io.fastapi.add_applications({"/ui": ui_app.index_page}, server=app)
+        for dir in ["css", "imgs"]:
+            app.mount(
+                f"/{dir}",
+                StaticFiles(directory=str(Path(__file__).parent / "_ui" / dir)),
+                name=dir,
+            )
```

Comment:
This should probably go into `BokehFastapi` under a `static_dirs` keyword similar to what `pn.serve` does.